packages/babel-parser/src/tokenizer/index.js

```diff
@@ -168,6 +169,21 @@ export default class Tokenizer extends LocationParser {
     return curr;
   }
 
+  // Get the next non-whitespace character position
+  lookaheadChPos(noNewLine?: boolean): number {
+    skipWhiteSpace.lastIndex = this.state.pos;
+    // $FlowIgnore
+    const [skip] = skipWhiteSpace.exec(this.input);
+
+    if (noNewLine && lineBreak.test(skip)) return -1;
+    return this.state.pos + skip.length;
+  }
+
+  lookaheadCh(noNewLine?: boolean): number {
```

Comment:
This does the same thing introduced by https://github.com/babel/babel/pull/10371 (when I initially wrote this code that PR didn't exist :stuck_out_tongue:)

I'll rebase this branch when that PR is merged, to use the method it introduces.